docs/plan-for-production.md

```diff
@@ -0,0 +1,345 @@
+<!--[metadata]>
++++
+title = "Plan for Swarm in production"
+description = "Plan for Swarm in production"
+keywords = ["docker, swarm, scale, voting, application,  plan"]
+[menu.main]
+parent="workw_swarm"
+weight=70
++++
+<![end-metadata]-->
+
+# Plan for Swarm in production
+
+This article provides guidance to help you plan, deploy, and manage Docker
+Swarm clusters in business critical production environments. The following high
+ level topics are covered:
+
+- [Security](#security)
+- [High Availability](#high-availability)
+- [Performance](#performance)
+- [Cluster ownership](#cluster-ownership)
+
+## Security
+
+There are many aspects to securing a Docker Swarm cluster. This section covers:
+
+- Authentication using TLS
+- Network access control
+
+These topics are not exhaustive. They form part of a wider security architecture
+that includes: security patching, strong password policies, role based access
+control, technologies such as SELinux and AppArmor, strict auditing, and more.
+
+### Configure Swarm for TLS
+
+All nodes in a Swarm cluster must bind their Docker Engine daemons to a network
+port. This brings with it all of the usual network related security
+implications such as man-in-the-middle attacks. These risks are compounded when
+ the network in question is untrusted such as the internet. To mitigate these
+risks, Swarm and the Engine support Transport Layer Security(TLS) for
+authentication.
+
+The Engine daemons, including the Swarm manager, that are configured to use TLS
+will only accept commands from Docker Engine clients that sign their
+communications. The Engine and Swarm support external 3rd party Certificate
+Authorities (CA) as well as internal corporate CAs.
+
+The default Engine and Swarm ports for TLS are:
+
+- Engine daemon: 2376/tcp
+- Swarm manager: 3376/tcp
+
+For more information on configuring Swarm for TLS, see the **need link to
+securing swarm article**
+
+### Network access control

```

Comment:
Agreed. I think we should whack a sentence before it though explaining that if you are using container networking then these re the ports....